legacy/firmware/stellar.c

```diff
@@ -481,6 +481,92 @@ bool stellar_confirmManageOfferOp(const StellarManageOfferOp *msg) {
   return true;
 }
 
+bool stellar_confirmManageBuyOfferOp(const StellarManageBuyOfferOp *msg) {
+  if (!stellar_signing) return false;
+
+  if (!stellar_confirmSourceAccount(msg->has_source_account,
+                                    msg->source_account)) {
+    stellar_signingAbort(_("Source account error"));
+    return false;
+  }
+
+  // Hash: operation type
+  stellar_hashupdate_uint32(3);
```

Comment:
this should be the BuyOffer operation type, 3 seems to be SellOffer

Reply:
Got it, thanks